core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/HealthMonitor.scala

```diff
@@ -94,6 +95,18 @@ class HealthMonitor private (val slickDataSource: SlickDataSource, val googleSer
   // not interfere with user facing operations.
   import context.dispatcher
 
+    /**
+    * A monoid used for combining SubsystemStatuses.
+    * Zero is an ok status with no messages.
+    * Append uses && on the ok flag, and ++ on the messages.
+    */
+  private implicit val SubsystemStatusMonoid = new Monoid[SubsystemStatus] {
+    def combine(a: SubsystemStatus, b: SubsystemStatus): SubsystemStatus = {
+      SubsystemStatus(a.ok && b.ok, a.messages |+| b.messages)
+    }
+    def empty: SubsystemStatus = OkStatus
+  }
+
```

Comment:
I was curious so I tried moving this back to the bottom, and it compiles for me.

See:
https://github.com/broadinstitute/rawls/compare/remove_workbenchlibs...rt-monoid-at-bottom
https://travis-ci.org/broadinstitute/rawls/builds/257961117

Not that I necessarily object to putting it at the top. :)